test/WireMock.Net.Tests/FluentMockServerTests.Proxy.cs

```diff
@@ -190,29 +193,30 @@ public partial class FluentMockServerTests
         [Fact]
         public async Task FluentMockServer_Proxy_Should_change_absolute_location_header_in_proxied_response()
         {
-            // given
-            _serverForProxyForwarding = FluentMockServer.Start();
+            // Assign
+            var settings = new FluentMockServerSettings { AllowPartialMapping = false };
+            _serverForProxyForwarding = FluentMockServer.Start(settings);
             _serverForProxyForwarding
                 .Given(Request.Create().WithPath("/*"))
                 .RespondWith(Response.Create()
                     .WithStatusCode(HttpStatusCode.Redirect)
                     .WithHeader("Location", _serverForProxyForwarding.Urls[0] + "testpath"));
```

Comment:
This creates a server that could redirect indefinitely I think. `Request.Create().WithPath("/")` instead?

Reply:
Fixed

Reply:
No, it has to be like this, else test fails.